### ISSUE TITLE
Looking for Sample Payload for pubsub transport which can be published from external systems

### ISSUE THREAD

**Issue by Vidya613** [rebus-org/Rebus#1174]
Can someone please provide me with Sample Payload for pubsub transport which can be published from external systems

**Comment by mookid8000** [comment]
Sorry, but I don't understand what you're asking for here 😅 
Could not maybe rephrase the question?

**Comment by Vidya613** [comment]
> we have created a solution with rebus saga handlers and gcp transport and and we have the subscriptions configured and in order this subscription to consume the payload published from other external systems what should be the format of the payoad if i have to publish from gcp pub sub. I guess i'm clear now.. Can anyone please help me with that.. For example i got this payload, i'm not sure if this is correct or not { "MessageType":"", "MessageData":{ "OrderId":12345, "BuyerId":67890, "TotalPrice":1000.0, "OrderItems":[{ "ProductId":1, "Count":10, "Price":100.0 }] }, "CorrelationId":"", "ReplyToAddress":"" }

@mookid8000
